package_control/ca_certs.py

```diff
@@ -213,19 +213,23 @@ def get_system_ca_bundle_path(settings):
 
 def ensure_ca_bundle_dir():
     """
-    Make sure we have a placed to save the merged-ca-bundle and system-ca-bundle
+    Make sure we have a place to save the merged-ca-bundle and system-ca-bundle
     """
 
     # If the sublime module is available, we bind this value at run time
-    # since the sublime.packages_path() is not available at import time
+    # since the sublime.cache_path() is not available at import time
     global ca_bundle_dir
 
     if not ca_bundle_dir:
-        ca_bundle_dir = os.path.join(sublime.packages_path(), 'User')
-    if not os.path.exists(ca_bundle_dir):
+        if int(sublime.version()) < 3000:
```

Comment:
I don't know about Will's policies here, but since this code is being reused here, it might make sense to outsource it as `get_cache_path` (or similar) into a shared module.

Reply:
Possible, but didn't want to add further modules in the first step. Efficiently it is required this way in 2 places, only, so didn't find it worth creating a shared module for a single function. http_cache.py needs some little different processing to ensure nested folders are created well.